docs/command-line-interface.md

```diff
@@ -0,0 +1,266 @@
+# OCI Runtime Command Line Interface
+
+This file defines the OCI Runtime Command Line Interface version 1.0.0.
```

Comment:
> Could you maybe add a sentence stating that this interface is specified as a test harness for compliance testing…

Done with b5deac1 → 4771073, borrowing from the very [similar][1] [approach][2] I'd taken in opencontainers/runtime-spec#513.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R4
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-958e7270f96f5407d7d980f500805b1bR12